app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -5,7 +5,7 @@
     <div>
       <h2 class="heading-m">Applications for teaching internships are now closed</h2>
 
-      <p>Our explore teaching advisers can let you know when applications re-open for the summer 2024 internships. They can also give helpful advice and insights about getting into teaching.</p>
+      <p>Our explore teaching advisers can let you know when applications re-open for the summer 2024 internships. They can also give free, helpful advice and insights about getting into teaching.</p>
```

Comment:
This looks fine but just to note that this banner may be updated prior to this PR being merged